code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm

```diff
@@ -184,6 +184,8 @@
 	xeno_owner.gain_plasma(drain_potency * 3.5)
 	xeno_owner.do_attack_animation(xeno_target, ATTACK_EFFECT_DRAIN_STING)
 	playsound(owner.loc, 'sound/effects/alien_tail_swipe1.ogg', 30)
+	xeno_owner.visible_message(message = span_xenowarning("\A [xeno_owner] stings [xeno_target]!"), self_message = span_xenowarning("We sting [xeno_target]!"), ignored_mob = xeno_target)
+	to_chat(xeno_target, span_userdanger("You're stung by [xeno_owner]!")) // We proudly tell our target they're fucked.
```

Comment:
Why, the player sees that they get stung and this doesnt give them any necessary long term info